src/main/java/data/Entry.java

```diff
@@ -76,6 +83,14 @@ public TimeSpan getEnd() {
     public TimeSpan getPause() {
         return pause;
     }
+    
+    /**
+     * If the entry is a vacation entry.
+     * @return If vacation entry.
```

Comment:
Should we rewrite the return to `@return True if entry is vacation, False otherwise`, just for clarification? Sorry that's excessively pedantic, but it crossed my way :)

Reply:
Well, I tried to adapt to the other JavaDocs ;) (But I fixed it now.)

> The end.